packages/cosmic-swingset/tools/inquisitor.mjs

```diff
@@ -721,29 +720,16 @@ const main = async (argv, options = {}, powers = {}) => {
 
   const { swingStore, mutations } = makeSwingStoreOverlay(argv[0]);
   const { db, kvStore } = swingStore.internal;
-  const fakeStorageKit = makeFakeStorageKit('');
-  const { toStorage: handleVstorage } = fakeStorageKit;
-  const receiveBridgeSend = (destPort, msg) => {
-    console.log('[bridge] received', msg);
-    switch (destPort) {
-      case BridgeId.STORAGE: {
-        return handleVstorage(msg);
-      }
-      default:
-        Fail`[inquisitor] bridge port ${q(destPort)} not implemented for message ${msg}`;
-    }
-  };
+  /** @type {Parameters<typeof makeCosmicSwingsetTestKit>[0]} */
   const config = {
-    swingsetConfig: { maxVatsOnline },
-    swingStore,
-    /** @type {Partial<SwingSetConfig>} */
-    configOverrides: {
+    fixupConfig: _config => ({
+      ..._config,
```

Comment:
We reserve underscore prefixes for unreferenced variables.
```suggestion
    fixupConfig: cfg => ({
      ...cfg,
```

P.S. I like letting `fixupConfig` subsume `configOverrides`.